composer.json

```diff
@@ -66,6 +66,10 @@
       "@phpcs-tests"
     ],
     "test-unit": "\"vendor/bin/phpunit\" --testsuite unit --color=always",
-    "test-integration": "\"vendor/bin/phpunit\" --testsuite integration --configuration tests/phpunit/integration/phpunit.xml.dist --color=always"
+    "test-integration": "\"vendor/bin/phpunit\" --testsuite integration --configuration tests/phpunit/integration/phpunit.xml.dist --color=always",
+    "test": [
```

Comment:
Hmm, great idea.  But the word "test" might have a future task assigned to it from a package or composer.  Instead, how about if we give it a unique task name?

Maybe something like:

```
composer run-tests
```

Therefore, the script then would be `"run-tests"`.

Reply:
Good point!
Should we brand it as 
`composer beans-tests` ?

Reply:
We could.  BUT it becomes less reusable at that point.  I like defining scripts that I can train my hands to run over and over again regardless of what package I'm in.

Let's go with `"run-tests"`.  Shoot, we can do the grouped sniffers too by saying: `composer run-phpcs`.

Reply:
Yes, keeping it consistent would be great.
Should I add the change to the PHPCS script or do you want a separate PR for that?

Reply:
Separate PR would be good since this one is specific for tests.